modules/mobile/spec/request/military_information_request_spec.rb

```diff
@@ -175,5 +175,16 @@
         expect(response).to have_http_status(:not_found)
       end
     end
+
+    context 'with a user not authorized' do
+      it 'returns a forbidden response' do
+        user = FactoryBot.build(:iam_user, :no_edipi_id)
+        iam_sign_in(user)
+        VCR.use_cassette('va_profile/post_read_service_history_403') do
```

Comment:
You should also remove this. It doesn't cause an error because the call is never made so VCR never tries to load it.